### ISSUE TITLE
Suggestion for the README (●'◡'●) -thanks for the other issue's popularity-

### ISSUE THREAD

**Issue by windowsmas** [TheTimeBunny/TimeBom#11]
Thanks for your contribution for that other issue I made ༼ つ ◕_◕ ༽つ

I've came up with an example README 
Here ya go!

### **TimeBom** (❁´◡`❁)
[insert timebom logo here]
TimeBom is a more friendly and open-source game engine developed by a team of contributors:

### Creator (Programmer, Artist, Sound FX, Designer, Publisher)
GlitchJustExists

### Contributors
bredcat999
redstone-scratch

We are excited to expand our project and are ambitious to be one of the best game engines!
Thanks,
The TimeBom Team :-)

**Comment by ajskateboarder** [comment]
A README should contain documentation. TBH, adding the contributors is a little redundant (it's shown on the right of the repo page)

**Comment by Sfreyjr** [comment]
> A README should contain documentation. TBH, adding the contributors is a little redundant (it's shown on the right of the repo page)

Correct. We currently don’t have much due to either limited access to GH, inability to get any meaningful work done, or due to the planning phase.

**Comment by redstone-dev** [comment]
> > A README should contain documentation. TBH, adding the contributors is a little redundant (it's shown on the right of the repo page)
> 
> Correct. We currently don’t have much due to either limited access to GH, inability to get any meaningful work done, or due to the planning phase.

Planning? Where are we planning this? It would be nice if the macOS devs had access to all of this cuz then we could port it a lot more easily.

**Comment by Sfreyjr** [comment]
> > > A README should contain documentation. TBH, adding the contributors is a little redundant (it's shown on the right of the repo page)
> > 
> > 
> > Correct. We currently don’t have much due to either limited access to GH, inability to get any meaningful work done, or due to the planning phase.
> 
> Planning? Where are we planning this? It would be nice if the macOS devs had access to all of this cuz then we could port it a lot more easily.
>
Apologies. I’ll get started on improvements to the README tonight.
